ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -165,6 +165,11 @@ public class KsqlConfig extends AbstractConfig {
   public static final String
       KSQL_SERVICE_ID_DEFAULT = "default_";
 
+  public static final String
```

Comment:
Since there's a new config, does it need to be added to the documentation?

Reply:
Good point Jim -- this is only for internal use but happy to add it to our doc if thats the policy (?)

Reply:
Depends on how this new config is going to be added to the auth plugins, I think. If it's going to be passed to a public auth interface then yes, we should definitely document it. I guess it has to be if it's going to be used for RBAC? RBAC code is always tricky since it's a separate plugin implementation outside the ksql repo, and the ksql repo only has the plugin interfaces and wiring.

Reply:
Thanks for the comments! Added some doc details [here](docs/reference/server-configuration.md) but I want to add more details when the RBAC plugin integration happens, for now we just need to push the new conf so we can populate it with our next release. Let me know if this makes sense